spec/test_track_spec.rb

```diff
@@ -275,5 +277,27 @@
         end
       end
     end
+
+    context 'when set via full assets:precompile command' do
+      around do |example|
+        Dir.chdir('spec/dummy') do
+          File.delete('testtrack/build_timestamp') if File.exist?('testtrack/build_timestamp')
+          example.run
+          system({ 'RAILS_ENV' => 'production' }, 'bundle exec rake assets:clobber')
```

Comment:
@aburgel technically this is more symmetric and ensures that any production-related assets get clobbered, but in practice it's slower than `rm`ing the `dummy/public/assets` dir. (This is why I had relied on `Rake::Task` in the prior implementation.)

But unless we feel like the test time is particularly painful for this repo (which it isn't) my vote would be to run the full rake command. Thoughts?

Reply:
I agree, especially if we plan to hook into `assets:clobber` to remove the file. We can just kill the following line.

Even better, we can move the clobber call into the test body and assert that the file is removed. Or is that overloading that test too much?